### ISSUE TITLE
"Pop out to new window" window size is not wide enough

### ISSUE THREAD

**Issue by patrickhlauke** [bitwarden/clients#12526]
### Steps To Reproduce

1. Open the extension
2. Use the "Pop out to a new window" button

### Expected Result

A sufficiently wide window is opened.

### Actual Result

The standalone window is marginally too narrow - buttons butt up right against the right-hand edge of the window, on the "Vault" tab the scrollbar for favourites/all items is not visible.

### Screenshots or Videos

![screenshot of the extension popped out into a standalone window](https://github.com/user-attachments/assets/db3bd3f7-2ea9-484f-97c5-cfda6af83f4c)

### Operating System

Windows

### Operating System Version

11

### Web Browser

Chrome

### Browser Version

131.0.6778.205 (Official Build) (64-bit)

### Build Version

2024.12.2

### Issue Tracking Info

- [x] I understand that work is tracked outside of Github. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by SergeantConfused** [comment]
Hello @patrickhlauke,

Thank you for your report. I was able to reproduce this and have flagged it to the Engineering department.
Please feel free to post additional information, such as screenshots or a screen video recordings, if you wish.

Thank you again,

**Comment by FaviFake** [comment]
A similar bug was posted on the community forum:

> The “Extra Wide” appearance option is not respected when you click on the arrow next to your profile icon to “Pop out to a new window”. So even though the new window has much better height for listing the vault elements, the width is narrower if you had the “Extra Wide” appearance option selected.

-- [Usability issues (UX) in redesigned UI (2024.12.0) - #828 by Tolga](https://community.bitwarden.com/t/usability-issues-ux-in-redesigned-ui-2024-12-0/76836/827#:~:text=The%20%E2%80%9CExtra%20Wide%E2%80%9D%20appearance%20option%20is%20not%20respected%20when%20you%20click%20on%20the%20arrow%20next%20to%20your%20profile%20icon%20to%20%E2%80%9CPop%20out%20to%20a%20new%20window%E2%80%9D.%20So%20even%20though%20the%20new%20window%20has%20much%20better%20height%20for%20listing%20the%20vault%20elements%2C%20the%20width%20is%20narrower%20if%20you%20had%20the%20%E2%80%9CExtra%20Wide%E2%80%9D%20appearance%20option%20selected)

**Comment by merissaacosta** [comment]
hey @patrickhlauke do you know if had default browser zoom settings on here?

**Comment by patrickhlauke** [comment]
@merissaacosta yes, default general "page zoom" in chrome://settings, and 100% zoom on the individual page itself

![snippet from the chrome settings, showing that page zoom is set to 100%](https://github.com/user-attachments/assets/69f4cfc1-ecc7-4206-9f21-aa6a9b61d118)

![bitwarden window, showing chrome's resize overlay showing it's set to 100%](https://github.com/user-attachments/assets/24618c2e-4087-40cd-9bed-59ec84fdbeb5)

**Comment by patrickhlauke** [comment]
I *think* this issue has now been resolved ... popping the current version out to a window doesn't have that awkward cut-off right-hand side anymore

![screenshot of bitwarden popup window, showing content correctly without cut-off](https://github.com/user-attachments/assets/147f9bf0-ee6f-4030-8567-1ded082e15c1)
